Client/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Lap 2 Code Challenge</title>
+    <script defer src="index.js"></script>
+    <link rel="stylesheet" href="style.css">
```

Comment:
Nice template! Looks like telegraph as well. Would be even better if you had a favicon and a more suitable name.